web/components/chat/ChatNameChangeMessage/ChatNameChangeMessage.tsx

```diff
@@ -27,7 +27,9 @@ export const ChatNameChangeMessage: FC<ChatNameChangeMessageProps> = ({ message
       </div>
       <div className={styles.nameChangeText}>
         <span style={{ color }}>{oldName}</span>
-        <span className={styles.plain}> is now known as </span>
+        <span className={styles.plain}>
+          <span id="owncast-name-change-text"> is now known as </span>
```

Comment:
Why is a new span required when one already existed that you could add an ID to?